engines/bmc.h

```diff
@@ -23,8 +23,10 @@ namespace pono {
 class Bmc : public Prover
 {
  public:
-  Bmc(const Property & p, smt::SmtSolver & solver);
-  Bmc(const PonoOptions & opt, const Property & p, smt::SmtSolver & solver);
+  Bmc(const Property & p, smt::SolverEnum se);
+  Bmc(const Property & p, smt::SmtSolver solver);
+  Bmc(const PonoOptions & opt, const Property & p, smt::SolverEnum se);
+  Bmc(const PonoOptions & opt, const Property & p, smt::SmtSolver solver);
```

Comment:
const ref to smtSolver

Reply:
Thanks, addressed with 0dbc624d1d5d9ab57c7977b758211620fdff5ecd